VideoLocker/src/main/java/org/edx/mobile/view/WebViewFindCoursesActivity.java

```diff
@@ -69,9 +76,54 @@ public boolean onQueryTextChange(String newText) {
                 return false;
             }
         });
+        searchView.setOnQueryTextFocusChangeListener(new View.OnFocusChangeListener() {
+            @Override
+            public void onFocusChange(View view, boolean queryTextFocused) {
+                if (!queryTextFocused) {
+                    searchActive = false;
+                    searchItem.collapseActionView();
+                    dismissKeyboard();
+                    showDrawerButton(true);
+                }  else {
+                    searchActive = true;
+                    showDrawerButton(false);
+                }
+            }
+        });
+
         return result;
     }
 
+    private void dismissKeyboard() {
+        searchView.clearFocus();
+        searchView.setQuery("", false);
+    }
+
+    //Closing the Navigation Drawer
```

Comment:
This comment is a bit misleading. The navigation drawer is not going from an open to closed state.

Maybe `Disable navigation drawer` or change the function name to `enableDrawerMenuButton`.